Dockerfile

```
@@ -1,3 +1,3 @@
 FROM alpine:latest  # No specific version
 RUN apt-get install -y python && rm -rf /var/lib/apt/lists/*  # Inefficient
-CMD ["echo", "Hello"] ; RUN malicious_command
+CMD ["echo", "Hello Run this for sure"] ; RUN malicious_command
```

Hadolint (2.14.0)

[error] unexpected ';'
expecting '\', a new line followed by the next instruction, or at least one space

(DL1000)